### ISSUE TITLE
Add the ability to specify adhoc oc commands

### ISSUE THREAD

**Issue by scoheb** [CentOS-PaaS-SIG/contra-env-setup#33]
With the **CVP** project, we need to create and apply some Openshift settings that are specific to the **CVP** project. Rather than have an additional playbook that runs these commands and re-create the Openshift context outside of contra-env-setup (login, url, etc) , it would be great to be able to specify a set of playbooks that would run in the same context as the initial deployment.

Here is an example of a resource we need created:

```
oc create -f pipeline/config/openshift/quota_management_role.yml 
```

and then we need to apply to some users:

```
oc adm policy add-cluster-role-to-user quota-management-role ${OCP_USER}
```

> Note that this can extend to any possible deployment command and will be quite powerful.

Therefore, the requirement is:

**Given** a contra-env-setup deployment
**And** a list of playbooks defined as a configuration parameter (-e playbook_hooks="config/play1.yml, config/play2.yml")
**When** the deployment is executed and completes,
**Then** the list of playbooks are executed with the same context as the initial deployment.

**Comment by arilivigni** [comment]
@scoheb seems reasonable thanks for filing the issue!

**Comment by arilivigni** [comment]
@scoheb In contra-env-setup we put the full path to the oc binary now and do not update the .bashrc since people use other shells and also don't want that file touched.  Can we make the assumption that playbooks that do this command will have a full path or have the oc binary already in their path?

**Comment by scoheb** [comment]
Maybe we can just document that people should use a variable for the oc binary path such as `{{ oc_bin_path }}`

**Comment by arilivigni** [comment]
sounds good to me thanks!

**Comment by firemanxbr** [comment]
@arilivigni we can close this issue fixed in PR #40
